docs/advanced.rst

```diff
@@ -1018,105 +1136,665 @@ to learn more about the use of this feature.
+|    :start-after: <!-->CONF-SERVER-CLIENT-PING<-->          |
+|    :end-before: <!--><-->                                  |
++------------------------------------------------------------+
+
+.. _DS_match_period:
+
+Server match period
+^^^^^^^^^^^^^^^^^^^
+
+As explained :ref:`above <DS_key_concepts>` the *Servers* received the hailing messages but they don't start at once to
+share publishers or subscribers info with the newcomers.
+They only trigger this process at regular intervals (match period).
+Note that this member is shared with the *client* setup but its name references solely the *client* functionality.
+This period is specified in the member:
+
+.. code-block:: bash
+
+    ParticipantAttributes.rtps.builtin.discovery_config.discoveryServer_client_syncperiod
+
+or the xml tag:
+
+.. code-block:: bash
+
+    dds>profiles>participant>rtps>builtin>discovery_config>clientAnnouncementPeriod
+
++------------------------------------------------------------+
+| **C++**                                                    |
++------------------------------------------------------------+
+| .. literalinclude:: ../code/CodeTester.cpp                 |
+|    :language: c++                                          |
+|    :start-after: //CONF_SERVER_SERVER_PING                 |
+|    :end-before: //!--                                      |
++------------------------------------------------------------+
+| **XML**                                                    |
++------------------------------------------------------------+
+| .. literalinclude:: ../code/XMLTester.xml                  |
+|    :language: xml                                          |
+|    :start-after: <!-->CONF-SERVER-SERVER-PING<-->          |
+|    :end-before: <!--><-->                                  |
++------------------------------------------------------------+
+
+.. _DS_setup_attributes:
+
+Server-client setup by attribute
+--------------------------------
+
+The settings related with server-client discovery are:
+
+.. csv-table::
+    :header: "Name", "Description"
+    :widths: 20,100
+
+    :ref:`RTPSParticipantAttributes <DS_RTPSParticipantAttributes>`, "Specifies general participant settings.
+    Some of them must be modified in order to properly configure a Server like the ``GuidPrefix``."
+    :ref:`BuiltinAttributes <DS_BuiltinAttributes>`, "It's a member of the above *RTPSParticipantAttributes* structure.
+    Allows to specify some mandatory server discovery settings like the :raw-html:`<br />` addresses were it listens for
+    clients discovery info."
+    :ref:`DiscoverySettings <DS_DiscoverySettings>`, "It's a member of the above *BuiltinAttributes* structure.
+    Allows to specify some mandatory client an optional server settings like the: :raw-html:`<br />` if it's a client or
+    a server or the list of servers it is linked to or the client-ping, server-match frequencies."
+
+.. _DS_RTPSParticipantAttributes:
```

Comment:
From here on, there is no new information, all was covered before. Instead of repeating the info it would be more maintainable to keep the settings attribute above and just add the links where each attribute/setting is explained in detail. This avoids duplicating the documentation (which may cause coherence issues in the future should the functionality change) while providing a table with a summary of applicable attributes for fast reference and links to the corresponding explanations at the same time.

Reply:
but this goes against Jaime's main directive: this docs section must be crystal clear. If we provide only conceptual or attribute centered documentations somebody will complain. In fact, at the beginning we have only an attribute centered one and nobody was happy about it.